lib/eslint-patch.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+var meld = require('meld');
+var docs = require('./docs');
+var Config = require("eslint/lib/config");
+var ConfigUpgrader = require('./config_upgrader');
+
+var supportedPlugins = ['react', 'babel'];
+
+module.exports = function patcher(eslint) {
+
+  meld.around(eslint.CLIEngine, 'loadPlugins', function(joinPoint) {
```

Comment:
@GordonDiggs @pointlessone Hey guys! I am trying to understand the reasons for this. Digging at the `eslint` source, it seems the `loadPlugings` function was removed on version 2.0, we are currently supporting version  3.14.1... is this a leftover from some experimentation?